continuous_integration/recipe/meta.yaml

```diff
@@ -18,16 +18,18 @@ build:
     - dask-sql-server = dask_sql.server.app:main
     - dask-sql = dask_sql.cmd:main
   string: py{{ python | replace(".", "") }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: RUST_BACKTRACE=full {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   build:
     - {{ compiler('rust') }}
-    - setuptools-rust >=1.4.1
+    - setuptools-rust >=1.5.2
+    - protobuf
```

Comment:
Think we should also add `protobuf` to our CI environments to explicitly make sure it's getting pulled, and maybe establish a min version like we do for `setuptools-rust`; can look into this a bit more

Reply:
So it looks like the package we want to pull in the `protoc` bin is `libprotobuf`, and based on the environment solve we probably want to pull in `libprotobuf=3` to make sure conflicts aren't raised with the `protobuf` package, can try this out locally and push up

Reply:
Actually looks like `protoc` was only needed for the Substrait components, so should be good to remove for now until we try to add in those components later on